apps/builder/app/builder/features/sidebar-left/help-popover.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Button,
+  Flex,
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+  theme,
+} from "@webstudio-is/design-system";
+import {
+  DiscordIcon,
+  GithubIcon,
+  TwitterIcon,
+  WebstudioWhiteIcon,
+  YoutubeIcon,
+} from "@webstudio-is/icons";
+import { useState, type ComponentProps } from "react";
+
+const PopoverItemButton = (
+  props: ComponentProps<typeof Button> & { background?: string; url?: string }
+) => (
+  <Button
+    css={{
+      background: props.background,
+      paddingTop: theme.spacing[5],
+      paddingBottom: theme.spacing[5],
+      paddingLeft: theme.spacing[7],
+      paddingRight: theme.spacing[7],
```

Comment:
I am surprised there is custom padding and not just directly usable button

Reply:
The default button's padding is too small. There's not enough breathing space.

Reply:
The button from the design system should not be changed. It was designed intentionally and should remain consistent with the rest of our UI that uses this same button.